Src/Support/Google.Apis.Auth.Tests/OAuth2/ExternalAccountCredentialTestsBase.cs

```diff
@@ -50,6 +50,7 @@ public abstract class ExternalAccountCredentialTestsBase
         protected const string ImpersonatedAccessToken = "fake_impersonated_access_token";
         protected const string QuotaProject = "fake_project_id";
         protected const string QuotaProjectHeaderName = "x-goog-user-project";
+        protected const string UniverseDomain = "fake.universe.domain.com";
```

Comment:
I think it would be clearer to call this FakeUniverseDomain, partly to make it more obvious that it's *not* a property in the initializer.